src/twisted/internet/test/test_inlinecb.py

```diff
@@ -145,3 +145,35 @@ def calling():
         d = calling()
         f = self.failureResultOf(d)
         self.assertIn("in erroring", f.getTraceback())
+
+    def test_forwardLotsOfTracebacks(self):
+        """
+        Several Chained inlineCallbacks gives information about all generators
+
+        Reproducing #9176
+        """
+
+        @inlineCallbacks
+        def erroring():
+            raise Exception()
+            yield "forcing generator"
```

Comment:
they way this is implemented now, leaves this line without coverage.

Can you do something like

```
yield "forcing generator"
raise Exception('some-error-marker')
```

In this way, the function should be converted into a generated and all the code is executed... and the marker text can be used in the assertion to make sure that the right error is raised, and this is not triggered by another error